docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5472,6 +5472,78 @@ the purpose of adding features for the P4Runtime API.
 | `@p4runtime_translation` | See sections [#sec-user-defined-types], [#sec-translation-of-port-numbers] |
 +--------------------------+---------------------------------------+
 ~
+### Custom Annotations in P4Info
```

Comment:
I think this should not be in this appendix, but instead moved to section 6 ("The P4Info Message"). The appendix was just meant as a way to quickly lookup which annotations are in the realm of P4Runtime.

Reply:
Agreed, will do.